### ISSUE TITLE
Add test for formatBytes optional parameter

### ISSUE THREAD

**Issue by shailpatels** [Submitty/Submitty#5457]
**What problem are you trying to solve with Submitty**
Add a test for the new optional parameter in the library function `formatBytes` in utils.php

Creating this issue and adding test as a separate PR so the Docker UI PR can be merged.

**Comment by MasterOdin** [comment]
While I recognize that we've been operating for our entire history on a principle of "we'll write tests later when we have time / there's not pressing bugs or issues / whatever", this has lead to the fact that tests are just not written (our PHP codebase is at 20% coverage, the python code coverage is equally poor). If we actually want our codebase to have tests, we need to start mandating that tests have to be included in PRs or the PR will not be merged, with really only an exception right now on changes to DatabaseQueries as we do not have a good way to test that. This is increasingly urgent to me as we've definitely hit a critical mass where it's hard to fully know about the scope of a change for both developer and reviewer, which is what unit tests will help us deal with. For example, the reason I opened #5484 was I discovered this while attempting to write tests for `site/app/controllers/EmailRoomSeating/Controller.php`. As a first step, I would be fine just saying that any changes to files under `app/libraries` and `app/models` requires a test as those should be largely stateless or require minimal mocking and then expand that to `app/controllers` after we've improved our testing docs for newcomers to the project.

I am also equally at fault of just writing code without testing as anyone.
